plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -549,6 +549,9 @@ protected String createTableSql(RemoteTableName remoteTableName, List<String> co
 
     protected String getColumnDefinitionSql(ConnectorSession session, ColumnMetadata column, String columnName)
     {
+        if (column.getComment() != null) {
+            throw new TrinoException(NOT_SUPPORTED, "This connector does not support creating tables with column comment");
```

Comment:
@ebyhr `getColumnDefinitionSql` method is used for both 'create table' and 'add column' flows
<img width="304" alt="image" src="https://user-images.githubusercontent.com/3051719/171381083-e672e523-31d9-4037-823f-71ebf3498bad.png">
So the exception message during 'add column' flow is not correct.
Could you please help to tackle that situation?

Reply:
Sure, let me fix.